examples/with-ava/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "backpack-examples-with-ava",
+  "version": "0.1.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "backpack",
+    "test": "ava -v",
+    "watch": "npm run test -- --watch"
+  },
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "backpack-core": "^0.2.0"
+  },
+  "devDependencies": {
+    "ava": "^0.18.2",
+    "babel-preset-env": "^1.1.8",
```

Comment:
backpack has preset-env built-in via `babel-preset-backpack`. No need to duplicate.